src/main/java/Duke.java

```diff
@@ -1,10 +1,154 @@
+import java.util.Scanner;
 public class Duke {
-    public static void main(String[] args) {
+
+    public static Task[] tasks = new Task[100];
+    public static int numTasks = 0;
+
+    public static void listTasks(){
+        System.out.println("Beep beep, listing out the tasks....Loading.....");
+        for(int i = 0; i < numTasks; i++){
+            System.out.println((i + 1) + "." + tasks[i].toString());
+        }
+    }
+
+    public static void addTask(String newTask){
+        tasks[numTasks] = new Task(newTask);
+        System.out.println("added: " + newTask);
+        numTasks += 1;
+        System.out.println("Beep boop, now you have " + numTasks + " tasks");
+
+    }
+
+    public static void addTodo(String newToDo){
+        tasks[numTasks] = new Todo(newToDo);
+        System.out.println("HELLO BEEP, added a new ToDo: ");
+        System.out.println("\t" + tasks[numTasks].toString());
+        numTasks += 1;
+        System.out.println("Beep boop, now you have " + numTasks + " tasks");
+    }
+
+    public  static void addDeadline(String param){
+        int deadlineIndex = param.indexOf("/by");
+        String description = param.substring(0, deadlineIndex);
+        String deadlineBy = param.substring(deadlineIndex + 4);
+        tasks[numTasks] = new Deadline(description, deadlineBy);
+        System.out.println("OH NO BEEP BEEP, a new Deadline: " + description);
+        System.out.println("\t" + tasks[numTasks].toString());
+        numTasks += 1;
+        System.out.println("Beep boop, now you have " + numTasks + " tasks");
+    }
+
+    public  static void addEvent(String param){
+        int deadlineIndex = param.indexOf("/at");
+        String description = param.substring(0, deadlineIndex);
+        String deadlineBy = param.substring(deadlineIndex + 4);
+        tasks[numTasks] = new Event(description, deadlineBy);
+        System.out.println("OH NO BEEP BEEP, a new Event: ");
+        System.out.println("\t" + tasks[numTasks].toString());
+        numTasks += 1;
+        System.out.println("Beep boop, now you have " + numTasks + " tasks");
+    }
+
+    public static void setTask(int whichTask, boolean done){
+        final String MESSAGE_DONE = "Nice! I've marked this task as done:";
+        final String MESSAGE_NOT_DONE = "OK, I've marked this task as not done yet:";
+        final String ERROR_OUT_OF_BOUND = "Sorry, the task does not seem to exist :<";
+        tasks[whichTask].setStatus(done);
+
+        if(whichTask > numTasks){
```

Comment:
Consider adding spaces before the opening parenthesis and after the closing parenthesis.